### DISCUSSION TITLE
how to use wireUI with local components

### DISCUSSION THREAD

**Discussion by rabol** [wireui/wireui#532] [Q&A] [answered]
Hi

How can. one use the WireUI component when Laravel Breeze is installed ?

e.g. <x-button> is also in /resources/views/components

how can I specify WireUI specific components ?

**Comment by ph7jack** [comment, marked as answer]
@rabol See this [customizing page](https://livewire-wireui.com/docs/customization).
You can rename the wireui components

**Reply by rabol** [reply]
At least on can rename all components - good workaround
other do something like <x-package.button ...>
